omnibus/config/software/inspec.rb

```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+#
+# Copyright 2015 Chef Software, Inc.
+#
+# All Rights Reserved.
+#
+
+name 'inspec'
+
+dependency 'ruby'
+dependency 'train'
+
+license "Apache-2.0"
+license_file "LICENSE"
+
+default_version "v0.18.0"
```

Comment:
[This is how chef does it](https://github.com/chef/chef/blob/master/omnibus/config/projects/chef.rb#L25-L27), but I suppose we could just `require_relative` lib/inspec/version.rb and use that...